src/main/kotlin/no/nav/dokgen/services/TemplateService.kt

```diff
@@ -338,6 +338,8 @@ class TemplateService @Autowired internal constructor(
         handlebars.registerHelper("divide", CustomHelpers.DivideHelper())
         handlebars.registerHelper("format-kroner", CustomHelpers.FormatKronerHelper())
         handlebars.registerHelper("trim-decimal", CustomHelpers.TrimDecimalHelper())
+        handlebars.registerHelper("array", CustomHelpers.ArrayHelper())
+        handlebars.registerHelper("inArray", CustomHelpers.InArrayHelper())
```

Comment:
Bruk navn `in-array` så det blir konsistent med øvrig navngivning

Reply:
Fikset